.gitignore

```diff
@@ -35,8 +36,11 @@ packageMounts/*
 _book/
 
 yarn.lock
-package-lock.json
 
 # Ignore old translation files
 locales/*/*_old.json
 locales/*/parameter.*.json*
+
```

Comment:
in theory, this should not be needed here, right?

Reply:
yes. Just I tried to test them.
So they were showed as new file. To be sure to not commit them.
Anyway it doesn't hurt